### PR TITLE
Update the help window

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

**Comment by huffman** [comment]
To test:

1. Confirm that you can see a new help window when going to `File > Help`

File: interface/resources/html/help.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<!-- Copyright 2015 High Fidelity, Inc. -->
```
Comment (zzmp) [inline interface/resources/html/help.html]: Copyright 2016

Comment (huffman) [inline interface/resources/html/help.html]: Fixed, thanks!

**Comment by hifi-gustavo** [comment]
Build console output for commit: 575d693a3cb7799362154d319c579058ae9c0915 available at: https://gist.github.com/00473b39eda3c58ed31192f35604ba34<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8745-575d693.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8745-575d693.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0e3e2b8a9a13293d98d2dd4926bce580d93a303a available at: https://gist.github.com/00473b39eda3c58ed31192f35604ba34<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8745-0e3e2b8.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8745-0e3e2b8.dmg

**Comment by zzmp** [comment]
In Keyboard + Mouse, under MORE INFORMATION, "doumentation"

**Comment by zzmp** [comment]
There is also significant lag between clicking a section and the image loading - can there be more immediate feedback?

**Comment by kevinmthomasse** [comment]
testing

**Comment by kevinmthomasse** [comment]
## QA Test
**QA Tester**: KevinMThomas
**QA Pass**: **QA Pass** SUCCESS
**Additional Test Results + Observations**:  Successfully confirmed that I saw a new help window when going to ```File > Help``` and screen shot enclosed.
capture

**Comment by huffman** [comment]
@zzmp I tested switching the tabs and I do notice a small delay that isn't noticeable when testing in a Chrome. I can change it to switch on mouse down which will be an improvement.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 246a816b804f557de57d974c9be45931590330c3 available at: https://gist.github.com/00473b39eda3c58ed31192f35604ba34<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8745-246a816.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8745-246a816.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e59069fe35370a326b7a3094a5575ed2e8a81705 available at: https://gist.github.com/00473b39eda3c58ed31192f35604ba34<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8745-e59069f.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8745-e59069f.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 508950841bb46faf459b34fad8624c939fc79702 available at: https://gist.github.com/00473b39eda3c58ed31192f35604ba34<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8745-5089508.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8745-5089508.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: a74b8f3f24786a90354169b7ac3f5a5e80746242 available at: https://gist.github.com/00473b39eda3c58ed31192f35604ba34<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8745-a74b8f3.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8745-a74b8f3.dmg

**Pull request merged**